x-pack/plugins/alerting/server/routes/rule_types.test.ts

```diff
@@ -111,6 +111,7 @@ describe('ruleTypesRoute', () => {
             "default_schedule_interval": "10m",
             "does_set_recovery_context": false,
             "enabled_in_license": true,
+            "has_get_summarized_alerts": undefined,
```

Comment:
Maybe we can add `hasGetSummarizedAlerts: true/false` on line 63 (listTypes const) and test its value here rather than undefined?